hdrop/README.md

```diff
@@ -1,37 +1,70 @@
 # hdrop
 
-This script emulates the main features of
-[tdrop](https://github.com/noctuid/tdrop) in Hyprland:
+This Bash script emulates the main features of [tdrop](https://github.com/noctuid/tdrop) in Hyprland:
```

Comment:
Does your editor automatically format the files like this or do you prefer them this way?

I've formatted them with `deno fmt` earlier, which does a similar job to `prettier`.

Reply:
I did it by hand to have a line break at 80 characters for the `--help` text and then just copied it over to readme.md. I now removed these manual line breaks in the readme file (i'm using prettier).

Do you mean to say that the new form is the "right" form and you're just wondering why it was different or is there some convention to insert manual line breaks after a certain number of characters?

Reply:
No, I mean when I run `deno fmt` it automatically wraps paragraphs at 80 chars, and your commit removed that wrapping. And I was wondering why?

Reply:
Prettier doesn't do that by [default](https://prettier.io/docs/en/options.html)
```
By default, Prettier will not change wrapping in markdown text since some services use a linebreak-sensitive renderer, e.g. GitHub comments and BitBucket. 
```
and i removed them by hand because i don't like manual line breaks.

I can tell `prettier` to wrap to 80 characters if you like but i'd rather not because GitHub will do line breaks on its own in the browser so i don't see the point.